src/main/java/org/folio/rest/helper/GroupFundFiscalYearHelper.java

```diff
@@ -41,4 +42,24 @@ public GroupFundFiscalYearHelper(HttpClientInterface httpClient, Map<String, Str
   public CompletableFuture<Void> deleteGroupFundFiscalYear(String id) {
     return handleDeleteRequest(resourceByIdPath(GROUP_FUND_FISCAL_YEARS, id, lang));
   }
+
+  public CompletableFuture<Void> updateBudgetIdForGroupFundFiscalYears(Budget budget) {
+    String fundId = budget.getFundId();
+    String fyId = budget.getFiscalYearId();
+    String query = "fundId==" + fundId + " AND fiscalYearId==" + fyId;
```

Comment:
Just pay attention that I've already implemented this in my PR as separate method https://github.com/folio-org/mod-finance/blob/95d6c08f1b7b9b21b6683176c73a78b4efdf1f30/src/main/java/org/folio/rest/helper/FundsHelper.java#L270